packages/styles/scss/components/modal/_modal.scss

```
@@ -292,6 +292,10 @@
       linear-gradient(to top, $layer 0, 2px, transparent 2px);
   }
 
+  .#{$prefix}--modal-scroll-content--no-fade {
+    mask-image: none !important; /* stylelint-disable-line declaration-no-important */
+  }
+
```

Will this add enough specificity to avoid the `!important`?

```suggestion
  .#{$prefix}--modal-scroll-content.#{$prefix}--modal-scroll-content--no-fade {
    mask-image: none;
  }

```

---

Yes thanks! I just had to add some more selectors to target for when it has the decorator/slug classes too for the AI-Label variant